index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+        <a href="#"><h1>
+              Sonu
+            </h1></a>
+            <desktop id="header-desktop">
+            <h2 id="portfolio">Portoflio</h2>
+            <h3 id="about-header">About</h3>
+            <h4 id="contact-header">Contact</h4>
+            <img src="images/letter-icon.svg" id="letter-icon">
+           <a href="#"><img src="images/Union.png" id="menu"></a>

```

Comment:
I suggest adding both the logo and this hamburger icon to a nav element `<nav>`.The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.